game.js

```diff
@@ -0,0 +1,148 @@
+let bot = require("./bot.js");
```

Comment:
cross refernce between files? wtf how it even works

Reply:
session variable announced in bot.js file, but she used in a game.js file too.

Reply:
it won't come in handy anymore